api/python/notebooks/api_demo/census_duplicated_cells.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Understanding and filtering out duplicate cells\n",
+    "\n",
+    "This tutorial provides an explanation for existence of duplicate cells in the Census, and it showcases different ways to handle these cells when performing queries on the Census using the `is_primary_data` cell metadata variable. \n",
+    "\n",
+    "**Contents**\n",
+    "\n",
+    "1. Why are there duplicate cells in the Census?\n",
+    "2. An example: duplicate cells in the Tabula Muris Senis data.\n",
```

Comment:
```suggestion
    "2. Examining duplicate cells in the Tabula Muris Senis data.\n",
```

Pretty much everything in the notebooks are examples, so suggesting this change.